### PR TITLE
Add navy orders, fixes to Soviet 06b

### PR DESCRIPTION
**Pull request by JovialFeline**
This changes Over the Bridge VizchGoi to add missing events, tweak how base rebuilds work, fix some oddities, and hopefully make the script a bit more straightforward to read.

----

The Allied navy has some "new" behavior that was in the original mission.

When the bridge is approached **or** 30 seconds pass, two Cruisers will shell part of the bridge until it breaks.
  - If approached first, there is also a camera reveal.
  - The bridge shelling is skipped on Easy difficulty.

https://github.com/OpenRA/OpenRA/assets/4985264/846a07dd-e9da-44f5-9641-2abfc6294192

Bridge_Triggers

If certain Allied buildings are attacked before the Radar Dome is captured or destroyed, a Cruiser moves toward the Soviet base to retaliate.
- This response is skipped on Easy difficulty.
- For better or worse, this is less effective without the original's obvious vision cheating.

RSPD_Triggers

----

The truck objective text correctly uses its translation.
- A fix for the same issue in Soviet 06a is included.

The Radar Dome's capture correctly stops Allied reinforcements.
- The Radar Dome is also checked before a wave is sent, rather than before the next wave is scheduled.

The Allies' rebuild logic has gone through some changes.
- When building, the area is first checked for obstructions.
  - If not clear, another attempt is scheduled.
  - If the obstruction is an idle Allied unit, they will be scattered.
- Buildings that are captured and sold can now be rebuilt (as a step toward https://github.com/OpenRA/OpenRA/issues/15356).
- ProduceArmor/ProduceInfantry restart when the War Factory/Allied Barracks is rebuilt.
  - Similar to https://github.com/OpenRA/OpenRA/issues/10617, whether the rebuilt structure will actually build units is down to luck at the moment. If the structure died in the middle of a Build, you can expect that queue to stay frozen.
- The rebuilding information and the rebuildable structure list are merged as a list of "blueprints".
  - Each blueprint location is now a plain CPos instead of a CVec added to a waypoint's CPos.

The Allies' unit production is slightly changed.
- ProduceArmor/ProduceInfantry is halted by a captured War Factory/Allied Barracks, instead of only a destroyed one.
- If the Allies' harvester dies, a replacement can be built.

Some unit facings that were mismatched from the original are fixed. This may be most noticeable on the main stretch of road where, on bleed, everything is pointed off to the side.

Facings

A tank and Artillery are nudged so they don't start on impassable terrain.

The Allies' starting artillery use Defend stance to mimic their original "sticky" orders.

The pre-placed "lazy" Allied units will start hunting when damaged. This seemed intended but the function for this was never called. Cruisers and artillery are skipped by this.

### CODE REVIEW

**Review by PunkPun — DISMISSED** [review]
LGTM, no idea what was done to lua though

File: mods/ra/maps/soviet-06b/map.yaml [hunk 1]
```diff
@@ -437,12 +441,12 @@ Actors:
...
 		Owner: Neutral
 		Location: 41,55
-	Barr: tent
+	Tent: tent
 		Location: 67,45
 		Owner: Greece
 	CameraStart: camera
 		Location: 46,61
-		Owner: Neutral
+		Owner:  Neutral
```
Comment (abcdefg30) [inline mods/ra/maps/soviet-06b/map.yaml]:
```suggestion
		Owner: Neutral
```

**Review by Smittytron — APPROVED** [review]
Tested and LGTM

**Comment by Smittytron** [comment]
[changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/0465ffba464275dbd596e368afe35249df68a25b)

**Pull request merged**